app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,123 @@
+* **Service ownership:** Keep track of your organization's service ownership by mapping teams and services. 
+* **Service health:** The Service Catalog consolidates key service health metrics into a single view, so you no longer need to check multiple tools (like GitHub and PagerDuty) for recent activity and status. It also helps you quickly identify and reuse existing services, reducing duplication and effort.
+* **Service governance and policy enforcement:** Enables governance in how services are created and maintained across your company to ensure you adhere to security, compliance, and engineering best practices.
+
+{% mermaid %}
+graph TD
+    subgraph "{{site.konnect_short_name}} Service Catalog"
+        SC[Service Catalog]
+        SG[{{site.base_gateway}} Service]
+        API[API Product]
+    end
+
+    subgraph "External APIs"
+        GitHub[GitHub]
+        PagerDuty[PagerDuty]
+        Data[External API Data]
+    end
+
+    GitHub -->|Pulls Service| SC
+    PagerDuty -->|Pulls Service| SC
+    SC -->|Maps to| SG
+    SC -->|Maps to| API
+    SC -->|Collects Data from| Data
+    Data -->|Reports to| SC
+{% endmermaid %}
+
+> Figure 2: This diagram shows how you can map services from one source, like GitHub, to another, such as a {{site.base_gateway}} service, in Service Catalog. Service Catalog also shows you how your services interact with external APIs.
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Service Catalog | The canonical system of record for your organization's services in {{site.konnect_short_name}}. This includes all the interactions between them and with external APIs. |
+| [Service](/gateway/latest/key-concepts/services/) | A service is the top-level entity in the Service Catalog. It typically represents an external upstream API or microservice that's owned by a singular team in your organization. For example, a data transformation microservice, a billing API, and so on. |
+| Integration | These are application integrations, either internal {{site.konnect_short_name}} applications or external applications. When enabled, an integration serves one of two purposes:<ul><li>Allows you to discover new Resources and subsequently create new services based on them. For example: creating a new {{site.konnect_short_name}} service called “Billing” from a GitHub Repository called “billing-repo-public”. In this case, the integration is GitHub and the Resource is the GitHub Repository called “billing-repo-public”.</li><li>Allows you to add helpful contextual information to a service that already exists in your catalog. For example: binding a {{site.base_gateway}} service called “trigger-payment” to a Service Catalog service. In this case, the integration is “Gateway Manager” and the Resource is the “trigger-payment” {{site.base_gateway}} service.</li> |
```

Comment:
We are in the process of writing Learning Hub content for Service Catalog, and as part of that, are including definitions for these terms. See here: https://docs.google.com/document/d/1JInLEJkm4D1y7RAFrfkTmG2AQNfy_vt_7hG1875-5yM/edit#heading=h.93cm9iglwi92

Let's make sure that we align on consistent definitions for both Kong Docs and Learning Hub.